classes/mail/Mailer.inc.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/**
+* @file classes/mail/Mailer.inc.php
+*
+ * Copyright (c) 2014-2021 Simon Fraser University
+* Copyright (c) 2000-2021 John Willinsky
+* Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Mailer
+ * @ingroup mail
+*
+ * @brief Represents interface to manage emails sending and view
+ */
+
+namespace PKP\mail;
+
+use Exception;
+use Illuminate\Contracts\Events\Dispatcher;
+use Illuminate\Contracts\Support\Htmlable;
+use Illuminate\Mail\Mailer as IlluminateMailer;
+use InvalidArgumentException;
+use PKP\i18n\PKPLocale;
+use Swift_Mailer;
+
+class Mailer extends IlluminateMailer
+{
+    const OPEN_TAG = '{';
+    const CLOSING_TAG = '}';
+    const DOLLAR_SIGN_TAG = '$';
+    /**
+     * @var null
+     * Don't bind Laravel View Service, as it's not implemented
+     */
+    protected $views = null;
```

Comment:
type hint?

Reply:
I deliberately set a null to the views here. It shouldn't have any other value. Keeping it just for documentation.